appservice/src/main/resources/static/vpn/sslrootcert.cer

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Why is this called a "root" certificate? Isn't this simply the SSL certificate for VPN? If so, the name ``ssl.cer`` would be less confusing.